spec/element_locator_spec.rb

```diff
@@ -192,7 +192,7 @@
         if Watir.prefer_css?
           expect_all(:css, "div").and_return(elements)
         else
-          expect_all(:xpath, ".//div").and_return(elements)
+          expect_all(:xpath, "(.//div)[contains(@class, 'oob')]").and_return(elements)
```

Comment:
I introduced the `()` in case the base xpath did any unions, etc. In other words, it can be applied to expressions like `.//div | .//span`, becoming `(.//div | .//span)[contains(...)]` instead of `.//div | .//span[contains(...)]`. My xpath is pretty rusty, though.